index.js

```diff
@@ -14,4 +13,29 @@ app.get('/', (req, res) => res.send("Hello"));
 function onListen() {
     console.log(`Listening on port ${port}`);
 }
-app.listen(port, onListen);
\ No newline at end of file
+app.listen(port, onListen);
+
+// connect to heroku database
+const { Client } = require('pg');
+
+const client = new Client({
+    host: 'ec2-18-233-32-61.compute-1.amazonaws.com',
+    port: 5432,
+    user: 'gadyrjddeeskwc',
+    password: '8a6d028082f3a13122125822711519bb0b547f597301b1e6006f89978c04e5aa',
+    database: 'd4i5njekim7upa',
+    ssl: {
+        rejectUnauthorized: false
```

Comment:
What's dis do?

Reply:
Also what's the benefit of using user/pass instead of the env var Heroku sets? Example of the other way [here](https://devcenter.heroku.com/articles/heroku-postgresql#connecting-in-node-js). Just thinking if the `host` name we're hardcoding changes then this might fail.

If we do use the url env variable way then looks like we should append "`sslmode=require`" to the end. Reference [here](https://devcenter.heroku.com/articles/connecting-to-heroku-postgres-databases-from-outside-of-heroku#ssl).

Reply:
I agree that env variable would be much cleaner, and also probably allow for easier/more modifiable testing if we choose to have multiple databases to test.